### ISSUE TITLE
Kappa pick has 31516 damage

### ISSUE THREAD

**Issue by whatthedrunk** [Kashdeya/Tiny-Progressions#167]
Guessing this was a mistake but wanted to make sure you knew. Thanks!
https://github.com/AllTheMods/ATM-5/issues/171

**Comment by GenDeathrow** [comment]
I thought it was mistake, but the original mod developer set that number. There is a config option to either turn it off amd/or change the damage value. Which is insanely high. I may look in reducing it for its default damage. but you should be able to change the damage.

**Comment by whatthedrunk** [comment]
So the issue is closed...? If so I gotta make the new recipe insane to balance it.

**Comment by GenDeathrow** [comment]
I finally figured out what had happen. Apparently this was an issue as far back as 1.12. The Configs said it was for durability but was linked up to dmg. Since I was just porting stuff over, I didin't noticed that. there was a switch. I kept looking at the code.. and was like no.. this is from the original mod. But the configs had a different wording than what the code showed. so I be changing the configs up to have a change. 

Edit: TBH Im not sure what was truly intended from the original developer. So I will just have to come up with some new numbers. I was only leaving the original numbers as I'm trying to not change too much of what was his mod. But I will be honest as that number looks more like a durability number instead of a damage number. I do know this was suppose to be the  strongest weapon, but it wasn't meant for all packs.  Also didn't realize the recipe was so easy. So.. I have no idea what the initial idea was for this.. other than it was some sort of streamer item.
